service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -279,6 +223,46 @@ public void notifyUnpaidOrder(Order order) {
         }
     }
 
+    private Long getAmountToPay(Order order) {
+        Long bonuses = order.getPointsToUse() == null ? 0L : order.getPointsToUse().longValue();
+        Long certificates = order.getCertificates() == null ? 0L
+            : order.getCertificates().stream()
+                .map(Certificate::getPoints)
+                .reduce(0, Integer::sum)
```

Comment:
can we use` Long::sum `here?

Reply:
No, because getPoints return Integer and map return Stream<Integer>